build.sbt

```diff
@@ -5,9 +5,9 @@ import com.typesafe.tools.mima.plugin.MimaKeys.{
   binaryIssueFilters, previousArtifacts
 }
 
-val PlayVersion = playVersion(sys.props.getOrElse("play.version", "2.5.4"))
+val PlayVersion = playVersion(sys.props.getOrElse("play.version", "2.5.8"))
```

Comment:
well here it should be `play.version` 2.5.9 as well, I guess, however I don't any place where you even use PlayVersion anymore.